README.md

```diff
@@ -3,3 +3,63 @@
 For testing purposes 3 x 300 clinical metadata, pipeline metadata and corresponding variants have been generated.
 
 Check releases for data files!
+
+## Building
+To build, you must have `wheel` installed. If you do not have it installed, run the following:
+
+`pip install wheel`
+
+Once you have installed `wheel`, run the following at the root level of this repository to build:
+
+`python setup.py sdist bdist_wheel`
+
+After this command has successfully run, a directory called `dist` should exist. Within this directory 
+there should be a `.whl` file and a `.tar.gz` file.
+
+## Installing
+To install, you must have `wheel` installed. If you do not have it installed, run the following:
```

Comment:
@amanjeev has mentioned in his earlier comments how he installed the package with `pip install -e .`, i think this should be documented, and possibly as a preferred way for local development, it's just easier since it doesn't require the installation of `wheel`.

Reply:
What exactly does the `-e` flag do? I know it means editable mode. but how is it different from just running `pip install` without the `-e` flag?

Reply:
@amanjeev can correct me, but `-e` is indeed the editable mode which gives you an editable installation for dev use, i believe `pip install -e .` is largely equivalent to `python setup.py develop`, while `pip install .` is equivalent to `python setup.py install`; so yeah the `-e` is more meant for development here i guess

Reply:
see https://stackoverflow.com/questions/19048732/python-setup-py-develop-vs-install for a more complete answer

Reply:
Sorry Jimmy, I misread your comment. Yes, you are correct. :)

Reply:
I am still a little confused. For the documentation, wouldn't it make more sense to write `pip install .` since we aren't expecting the user to do any development?

Reply:
Good question! I think in usage, we should do `pip install package` and under instructions for development we can say how to install as editable, how to test etc. What do you think of this?

Reply:
Yeah, I think that is a good idea. I don't have a Development section however. Where should I add that section?